### ISSUE TITLE
Python Missing

### ISSUE THREAD

**Issue by XGavinp** [yoshi389111/github-profile-3d-contrib#79]
Hi,

I really love this feature and have implemented it in my own README but fir some reason it does not show Python in the Pie Chart. Can u help me solve this issue. https://github.com/XGavinp

**Comment by NarciSource** [comment]
This pie chart doesn't actually reflect your overall language usage; rather, it only calculates the primary language for each repository.
I think Python doesn't show up in the chart because it appears as the primary language in only a few of your repositories.

**Comment by XGavinp** [comment]
But all the other languages show.

**Comment by Li-can-cheng** [comment]
Can we solve this problem together

**Comment by lgmorand** [comment]
> But all the other languages show.

it's takes only the first top 5 langages.  you have to edit this line

```
const languages = userInfo.contributesLanguage.slice(0, 5);
```

**Comment by XGavinp** [comment]
Where do I edit this line?

**Comment by lgmorand** [comment]
Either you are lazy (there are maximum 10 ten small files to open), or you don't know how to use the search input on top of this page buddy :) Either way, I'm sure you will find it :)

**Comment by XGavinp** [comment]
I have searched all the files and have not found it.

**Comment by lgmorand** [comment]
your are looking for changing langages in the pie chart, and there is a file named "https://github.com/yoshi389111/github-profile-3d-contrib/blob/main/src/create-pie-language.ts"

**Comment by XGavinp** [comment]
Yes But this is in the Owners Repo I can't change it without access.

**Comment by lgmorand** [comment]
of course, as all github projects. you fork it and do your own version or you can do a pull request. that's basics

**Comment by yoshi389111** [comment]
@XGavinp
Thank you for your report.

Currently the language pie chart only shows the top 5 languages ​​by contribution.

So languages ​​after 6th are not shown.

Due to limited display space, we need to limit the number of languages ​​shown, and I think the top 5 is a good number in that case.
